private/rules/coursier.bzl

```diff
@@ -726,27 +726,45 @@ def infer_artifact_path_from_primary_and_repos(primary_url, repository_urls):
             break
     return primary_artifact_path
 
+def _artifact_to_coordinate(artifact):
+    """
+    Convert an artifact to a maven coordinate string.
+    """ 
+
+    return artifact["group"] + ":" + artifact["artifact"] + (":%s" % artifact["classifier"] if artifact.get("classifier") != None else "")
```

Comment:
The coordinate handling code is now in `//private/lib:coordinates.bzl`. You likely want to use `to_external_form` here, or perhaps add an additional function.